terraform/modules/happy-mesh-access-control/main.tf

```diff
@@ -42,7 +43,7 @@ resource "kubernetes_manifest" "linkerd_mesh_tls_authentication" {
         "kind"      = "ServiceAccount"
         "name"      = "nginx-ingress-ingress-nginx"
         "namespace" = "nginx-encrypted-ingress"
-      }] : [])
+      }] : [], global_allow_list)
```

Comment:
can you pull this terinary out into a local variable and name it? took me a couple reads to understand how it was working. I didn't actually know that you could company lists of objects like that. Mind adding a `terraform console` output showing the comparision works